### PR TITLE
feat: CFD-299 View Purchase Methods

### PR DESCRIPTION
**Pull request by jacsamell**
# Description

-   View purchase methods in global setting

# Testing instructions

-   Check you can view previously created SOPs in global settings
- NOTE: edit/delete/create will not work yet.

# Type of change

-   [X] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

File: repos/fdbt-site/src/layout/SubNavigation.tsx [hunk 1]
```diff
@@ -28,19 +28,19 @@ const SubNavigation = (): ReactElement => {
                         </a>
                     </li>
 
-                    <li className="app-subnav__section-item">
+                    {/*<li className="app-subnav__section-item">
```
Comment (RobCatton) [inline repos/fdbt-site/src/layout/SubNavigation.tsx:31]: Is this to be deleted?

File: repos/fdbt-site/src/layout/SubNavigation.tsx [hunk 2]
```diff
@@ -55,23 +55,23 @@ const SubNavigation = (): ReactElement => {
...
                         <a
```
Comment (RobCatton) [inline repos/fdbt-site/src/layout/SubNavigation.tsx:59]: And this

File: repos/fdbt-site/src/layout/SubNavigation.tsx [hunk 3]
```diff
@@ -55,23 +55,23 @@ const SubNavigation = (): ReactElement => {
...
                             href="/globalSettings"
                         >
                             Multi-operator groups
                         </a>
-                    </li>
+                    </li>*/}
 
-                    <li className="app-subnav__section-item">
+                    {/*<li className="app-subnav__section-item">
                         <a
```
Comment (RobCatton) [inline repos/fdbt-site/src/layout/SubNavigation.tsx:68]: And this

File: repos/fdbt-site/src/layout/SubNavigation.tsx (hunk 1 above)
Comment (jacsamell) [reply repos/fdbt-site/src/layout/SubNavigation.tsx:31]: They'll be added in again as we add the other global settings, I did toy with the idea of deleting them as they will be easy to re-add. What do you think?

File: repos/fdbt-site/tests/components/GlobalSettingsViewPage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { GlobalSettingsViewPage } from '../../src/components/GlobalSettingsViewPage';
+
+const CardBody = ({ entity: { id, name } }: { entity: { id: number; name: string } }) => (
```
Comment (RobCatton) [inline repos/fdbt-site/tests/components/GlobalSettingsViewPage.test.tsx:5]: should this be cardBody?

**Review by RobCatton — APPROVED** [review]

File: repos/fdbt-site/tests/components/GlobalSettingsViewPage.test.tsx (hunk 4 above)
Comment (jacsamell) [reply repos/fdbt-site/tests/components/GlobalSettingsViewPage.test.tsx:5]: As discussed it's react component

**Pull request merged**